EventFilter/RPCRawToDigi/plugins/RPCCPPFUnpacker.cc

```diff
@@ -217,7 +220,7 @@ void RPCCPPFUnpacker::processRXRecord(RPCAMCLink link,
     return;
   }
 
-  if (bx < bx_min || bx > bx_max) {
+  if ((bx - cppfDaq_Delay) < bx_min || (bx - cppfDaq_Delay) > bx_max) {
```

Comment:
wouldn't it make sense to define e.g. 

```
auto bx_corrected = bx - cppfDaq_Delay;
```
and use it everywhere where the shifted value must be used? does the meaning of bx==0 change or stay the same?

Reply:
Thanks @jpata 
> does the meaning of bx==0 change or stay the same
Good question! It looks that is needed for the DPG analysis, but could be deeper. Let me have some time to debug and investigate.

Reply:
auto bx_corrected implemented as suggested.

Reply:
Hi @jpata !
I have updated #38979, as well.

On the question "does the meaning of bx==0 change or stay the same".
I checked and it does change nothing both in the CPPF DAQ data (attached cppf_rpc_daq_bx_check.png) or in the input to EMTF (l1_L1Stage2EMTF_rpcHitBX.png). 
So, we would prefer to not touch it for the moment.
![cppf_rpc_daq_bx_check](https://user-images.githubusercontent.com/5401038/183610156-f7674b68-a203-4d43-956f-ae257d54307d.png)
![l1_L1Stage2EMTF_rpcHitBX](https://user-images.githubusercontent.com/5401038/183610201-ffee1533-c8de-4f80-915d-9a0a22827e58.png)

In order to test I used 136.897_RunCosmics2021CRUZET workflow, but I changed the input data. The default run for this workflow is 344518, however in this run the CPPF key were not enabled to EMTF. (I.e. - no RPC input to EMTF, just DAQ data.) Because of this I changed with another cosmics run 356088.
runTheMatrix test ran without problems.

However, I decided to run with more events - the default are 100 events.
And on the 102th event the Module: L1TStage2CPPF:l1tStage2Cppf crashed with segmentation violation.
But I don't I am not sure the crash is caused of our update,
as the same module already was reported to crash with release validation, further to latest comments in #38564 .
ping @ram1123 and @zhangcg123 

I can give more details during the meeting.